src/platform/forms-system/src/js/fields/FileField.jsx

```diff
@@ -107,14 +107,15 @@ class FileField extends React.Component {
       if (currentFile.type === 'testing') {
         // Skip read file for Cypress testing
         checkResults = {
-          checkTypeAndExtensionMatches: () => true,
-          checkIsEncryptedPdf: () => false,
+          checkTypeAndExtensionMatches: true,
+          checkIsEncryptedPdf: false,
```

Comment:
This is the bug that needed fixing. Line 134/135 (expand the code below) is expecting a boolean value, not a function.